spec/support/test_factories.rb

```diff
@@ -1,22 +1,29 @@
 module TestFactories
 
   def associated_post(options={})
-      post_options = {
-        title: 'Post Title', 
-        body: 'Post bodies must be pretty long',
-        topic: Topic.create(name: 'Topic name'),
-        user: authenticated_user
-      }.merge(options)
+    post_options = {
+      title: 'Post Title', 
+      body: 'Post bodies must be pretty long',
+      topic: Topic.create(name: 'Topic name'),
+      user: authenticated_user
+    }.merge(options)
 
-      Post.create(post_options)
-    end
+    Post.create(post_options)
+  end
 
-    def authenticated_user(options={})
-      user_options = {email: "email#{rand}@fake.com", password: 'password'}.merge(options)
-      user = User.new(user_options)
-      user.skip_confirmation!
-      user.save
-      user
-    end
-  
+  def authenticated_user(options={})
+    user_options = {email: "email#{rand}@fake.com", password: 'password'}.merge(options)
```

Comment:
Line is too long. [88/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.